deployment/Triton/README.md

```diff
@@ -0,0 +1,277 @@
+
+## User Documentation
+
+* [Quick Start](#quick-start)
+* [Examples](#examples)
+* [Usage](#usage)
+* [Model Config File](#model-config-file)
+* [Error Hanldling](#error-handling)
+
+
+## Quick Start
+
+1. Build Triton Container Image and Copy Model repository files using shell script
+
+```
+$ ./triton_build.sh
+```
+2. Run Triton Container Image in Background Terminal using provided shell script
+The supplied script will start the demo container with Triton and expose the three ports to localhost needed for the application to send inference requests. 
+```
+$ ./triton_run_local.sh
+```
+3. Install environment for client
+The client environment should have Python 3 installed and should have the necessary packages installed. 
+```
+$ python3 -m pip install -r requirements.txt
+```
+4. Other dependent libraries for the Python Triton client are available as a Python package than can installed using pip
+```
+$ pip install nvidia-pyindex
+$ pip install tritonclient[all]
+```
+5. Run the client program
+The [client](./client/client.py) program will take an optional file input and perform  classification to determine whether the study shows COVID-19 or not COVID-19.  See the [NVIDIA COVID-19 Classification ](https://ngc.nvidia.com/catalog/models/nvidia:med:clara_pt_covid19_3d_ct_classification) example in NGC for more background. 
+```
+$ python -u client/client.py [filename/directory]
+```
+and the program returns
+``` Default input for the client is client/test_data/prostate_24.nii.gz
+$ Classification result: ['Prostate segmented']
+```
+## Examples:
+The example demonstrates running a Triton Python Backend on a single image classification problem.  
+1. First, a Dockerfile and build script is used to build a container to Run the Triton Service and copy the model specific files in the container. 
+```Dockerfile:
+# use desired Triton container as base image for our app
+FROM nvcr.io/nvidia/tritonserver:21.04-py3
+
+# create model directory in container
+RUN mkdir -p /models/monai_covid/1
+
+# install project-specific dependencies
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+RUN rm requirements.txt
+
+# copy contents of model project into model repo in container image
+COPY models/monai_covid/config.pbtxt /models/monai_covid
+COPY models/monai_covid/1/model.py /models/monai_covid/1
+COPY models/monai_covid/1/covid19_model.ts /models/monai_covid/1
+
+ENTRYPOINT [ "tritonserver", "--model-repository=/models"]
+#ENTRYPOINT [ "tritonserver", "--model-repository=/models", "--log-verbose=1"]
+```
+Note: The Triton service expects a certain directory structure discussed in [Model Config File](#model-config-file) to load the model definitions.
+
+2. Next, the container with the Triton Service is run as a service (in background or separate terminal for demo).  In this example, the ports used by the Triton Service are set to `8090` for client communications.
```

Comment:
`is run` -> `runs`?

Thanks.

Reply:
excellent catch.  Hopefully, the modifications are more readable.
`2. Next, the container with the Triton Service runs as a service (in background or separate terminal for demo).  In this example, the ports used by the Triton Service are set to `8000` for client communications.`